### PR TITLE
Add number lookup sample

### PR DESCRIPTION
**Pull request by ericasp16**
## Purpose

Add samples for the number lookup preview

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
see the README in the pull request

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: lookup-phone-numbers-quickstart/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Before running sample code
+
+1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory that you'd like to clone the sample to.
+2. `git clone https://github.com/Azure-Samples/communication-services-python-quickstarts.git`
+3. With the Communication Services procured in pre-requisites, add connection string as an environment variable named `COMMUNICATION_SERVICES_CONNECTION_STRING`
+4.  Decide which lookup you would like to perform, and keep in mind that looking up all the operator details incurs a cost, while looking up only number formatting is free.
+
+> [!WARNING]
+> If you want to avoid incurring a charge, comment out lines 22-32
```
Comment (guoqing2023) [inline lookup-phone-numbers-quickstart/README.md:39]: lines 25-40?

**Review by guoqing2023 — APPROVED** [review]

**Review by vac0224 — APPROVED** [review]
LGTM

**Review by vac0224 — APPROVED** [review]
Approve

**Review by vac0224 — APPROVED** [review]
LGTM

**Pull request merged**